front/src/services/fetchServer.js

```diff
@@ -0,0 +1,5 @@
+import store from '@store';
+
+export const fetchServer = () => {
+  store.dispatch('userSettings/loadUserSettings');
```

Comment:
why did you do it? it is really confused me

Reply:
In future, here we can add actions with the same functionality